packages/react/test/errorboundary.test.tsx

```diff
@@ -22,10 +22,10 @@ const TestApp: React.FC<ErrorBoundaryProps> = ({ children, ...props }) => {
   return (
     <ErrorBoundary
       {...props}
-      onReset={(err: Error, stack: string) => {
+      onReset={(...args) => {
         setError(false);
         if (props.onReset) {
-          props.onReset(err, stack);
+          props.onReset(...args);
```

Comment:
Make sure we pass all the arguments, instead of having to edit this all the time when `onReset` changes.